styles/colors-silicon-day.less

```diff
@@ -298,6 +302,17 @@
 @agate-switch-focus-color: black;
 @agate-switch-label-color: @agate-accent;
 
+// ThumbnailItem
+// ---------------------------------------
+@agate-thumbnailitem-border-color: fade(@agate-black, 40%);
+@agate-thumbnailitem-focus-bg-color: fade(@agate-black, 15%);
+@agate-thumbnailitem-selected-bg-color: transparent;
+@agate-thumbnailitem-subcontent-text-color: #999999;
+@agate-thumbnailitem-subcontent-focus-text-color: #999999;
```

Comment:
Please define a less variable for #999999 and use it here.

Reply:
How about defining #999999 to @agate-noble? https://www.htmlcsscolor.com/hex/999999 says #999999 color is noble.

Reply:
Interesting. "nobel" never heard that before. I would be in favor of `@silicon-nobel-gray`. At least it's more quickly recognizable as a color that way. ;)